### ISSUE TITLE
The add-on appears on every add-on panel

### ISSUE THREAD

**Issue by verekia** [react-theming/storybook-addon-material-ui#84]
Hey! Thanks for your great work on the plugin.
I'm not sure if I'm doing something wrong here, but the add-on seems to appear on every panel, under the rest:

<img width="1144" alt="screenshot 2018-09-13 13 03 25" src="https://user-images.githubusercontent.com/522007/45466584-a6013400-b755-11e8-9482-ec1a1e1449f6.png">

I am using:
```
@babel: 7.0.0
@material-ui/core: 3.0.3
@storybook/addons: 4.0.0-alpha.21
@storybook/react: 4.0.0-alpha.21
storybook-addon-material-ui: 0.9.0-alpha.14
```

Cheers :)

**Comment by UsulPro** [comment]
Hi @verekia ))) Thank you!
Hm... it's really strange behavior. In general, Storybook should handle panels switching. I need to investigate what could be broken.

**Comment by UsulPro** [comment]
fixed in `0.9.0-alpha.15`

**Note:** all other addons should be compatible with `Storybook v4` as well
